### ISSUE TITLE
TODO???

### ISSUE THREAD

**Issue by Taki21** [applqpak/BasicForum#1]
Im a little lost xD! Can u help me out? Like what can i help on? Also, which service will u host on?

**Comment by remotevase** [comment]
Yeah, I dont know what is going on here yet...

**Comment by applqpak** [comment]
Add support for threads/categories? ;P I've done the bit where you can post(and, see the posts) but they're not in any specific order/placement. It should be able to allow the user to create a thread and post there.

**Comment by Taki21** [comment]
And it isnt allowing? Wait,,, Dont we need a database though?

**Comment by remotevase** [comment]
yeah...

**Comment by remotevase** [comment]
Isnt that part of config.php?

**Comment by Taki21** [comment]
can i do emojis?

**Comment by remotevase** [comment]
Not yet, That is a relatively advanced function

**Comment by Taki21** [comment]
not really i was thinking of using str_replace :)

**Comment by remotevase** [comment]
Yes, that is simple, but they gotta replace the text with an icon, then in order to have an icon in a post, there must be a database to upload the icons to.

**Comment by dragonwocky** [comment]
Why not just store the icons as images in one of the directories? That would make adding support for custom themes and other extensions later way easier and you wouldn't need a database...

**Comment by applqpak** [comment]
You need a database to store the messages lol.

**Comment by dragonwocky** [comment]
Oh lol

**Comment by remotevase** [comment]
Well... How's it going now?

**Comment by lavender-sys** [comment]
For the emojis you can use EmojiOne. 
@Taki21

**Comment by Taki21** [comment]
wut
